content/uri-consolidation/index.md

```diff
@@ -0,0 +1,63 @@
+---
+rfc:
+start_date: 2018-03-08
+pr:
+status: draft
+---
+
+# URI Consolidation
+
+## Summary
+
+This RFC proposes a change in the REST API to consolidate endpoints of the
+same resource type to use the same naming convention so CURIEs can be used to
+refer to the full collection of records within a Register.
+
+
+## Motivation
+
+Currently the REST API uses the plural form of a resource type for collection
+endpoints and the singular form for collection elements. Based on how Registers
+use CURIEs to reference records this design choice makes it impossible to
+refer to the whole list of records using the same CURIE prefix.
+
+
+## Explanation
+
+To address this limitation, we have two options: use the same form for the
+endpoints, or define two prefixes per register. Using the latter option would
+make defining and evolving registers more complex than using the former
+option, so we propose to only use the plural form for endpoints.
+
+We selected the plural form because it's more common in well-known APIs (e.g.
+[GitHub API v3](https://developer.github.com/v3)).
+
+Endpoint mappings:
+
+* From `/record/{field-value}` to `/records/{field-value}`.
+* From `/entry/{entry-number}` to `/entries/{entry-number}`.
+* From `/item/{item-hash}` to `/items/{item-hash}`.
+
```

Comment:
We could also add the below. It's not documented yet but the endpoint exists in the implementation and for two beta registers.

From `/index/{index-name}/record/{key}` to `/index/{index-name}/records/{key}`

Reply:
As discussed we will address this when it's formally introduced to the model and API.